### PR TITLE
fix test/e2e/network-02

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
use log functions of core framework

**Which issue(s) this PR fixes**:

Ref #81427

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/network/util_iperf.go [hunk 1]
```diff
@@ -23,8 +23,6 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
-
-	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
```
Comment (s-ito-ts) [inline test/e2e/network/util_iperf.go]: "k8s.io/kubernetes/test/e2e/framework" is needed to use framework.Logf and framework.Failf.

Comment (hwdef) [reply test/e2e/network/util_iperf.go]: Thanks for your suggestion.I changed it.

**Pull request closed without merging**